screen/reports/src/main/java/com/ivy/reports/ReportFilter.kt

```diff
@@ -18,7 +18,9 @@ data class ReportFilter(
     val maxAmount: Double?,
     val includeKeywords: List<String>,
     val excludeKeywords: List<String>,
-    val selectedTags: List<TagId>
+    val includedTags: List<TagId>,
+    val excludedTags: List<TagId>,
```

Comment:
Why do we need both? If the tag isn't selected => it's excluded? We don't need to maintain two lists

Reply:
Maybe to get all transactions except a few,

Useful when you have a lot of tags!

Reply:
Makes sense. Let's keep it then